lib/kele.rb

```diff
@@ -15,13 +15,37 @@ def initialize(email, pass)
   end
 
   def get_me
-    response = self.class.get('/users/me', headers: { authorization: @auth_token })
+    response = self.class.get('/users/me', headers: auth_header)
     JSON.parse(response.body)
   end
 
   def get_mentor_availability(mentor_id)
-    response = self.class.get("/mentors/#{mentor_id}/student_availability", headers: { authorization: @auth_token})
+    response = self.class.get("/mentors/#{mentor_id}/student_availability", headers: auth_header)
     JSON.parse(response.body)
   end
 
+  def get_messages(page_number = nil)
+    if page_number != nil
+      @page = {page: page_number}
+      response = self.class.get('/message_threads', headers: auth_header, body: @page)
```

Comment:
If you set this outside your if statement, you can just write it once and add body as an attribute.